posthog/utils.py

```diff
@@ -548,10 +549,10 @@ def get_instance_realm() -> str:
     """
     if settings.MULTI_TENANCY:
         return "cloud"
-    elif os.getenv("HELM_INSTALL_INFO", False):
+    elif is_clickhouse_enabled():
         return "hosted-clickhouse"
     else:
-        return "hosted"
+        return "hosted-postgres"
```

Comment:
I'm not sure we should change this. While it certainly makes it clearer, it will affect all historical data and in my past experience, when doing analytics it's so easy to forget about these changes that ends up being quite messy. Perhaps we should just add a comment in the code so whenever we go back to check this, it's clear that this represents Postgres?

Reply:
We can set it to a constant that has the better naming but the underlying value can remain the same?